src/main/java/housing/Region.java

```diff
@@ -1,10 +1,17 @@
 package housing;
 
+import collectors.HouseholdStats;
+import collectors.HousingMarketStats;
```

Comment:
Where are these two new imports used?

Reply:
As I mentioned before, it is because I used it previously, but it was not removed automatically in the Section of import when I deleted the code in the main body.

Reply:
Ok, no worries.